examples/stylesheet.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Here is an example stylesheet. Put it in ~/.config/ami.

Reply:
This looks good!